src/microbit/model/constants.py

```diff
@@ -0,0 +1,16 @@
+BLANK = "00000:00000:00000:00000:00000:"
+BOAT = "05050:05050:05050:99999:09990:"
+BRIGHTNESS_ERR = "brightness out of bounds"
+COPY_ERR_MESSAGE = "please copy() first"

```

Comment:
Are we considering internationalization?
Hard coding these messages, would make i18n a bit difficult in the future.

Reply:
We are likely going to deal with optimizing for localization later, since we're not sure if we'll have time for it.